docs/swarm/how-swarm-mode-works/nodes.md

```diff
@@ -0,0 +1,94 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/swarm/how-swarm-mode-works/"
+]
+title = "How nodes work"
+description = "How swarm nodes work"
+keywords = ["docker, container, cluster, swarm mode, node"]
+advisory = "rc"
+[menu.main]
+identifier="how-nodes-work"
+parent="how-swarm-works"
+weight="3"
++++
+<![end-metadata]-->
+
+# How nodes work
+
+Docker Engine 1.12 introduces swarm mode that enables you to create a
+cluster of one or more Docker Engines called a swarm. A swarm consists
+of one or more nodes: physical or virtual machines running Docker
+Engine 1.12 or later in swarm mode.
+
+There are two types of nodes: [**managers**](#manager-nodes) and
+[**workers**](#worker-nodes).
+
+![Swarm mode cluster](../images/swarm-diagram.png)
+
+If you haven't already, read through the [swarm mode overview](../index.md) and [key concepts](../key-concepts.md).
+
+## Manager nodes
+
+Manager nodes handle cluster management tasks:
+
+* maintaining cluster state
+* scheduling services
+* serving swarm mode [HTTP API endpoints](../../reference/api/index.md)
+
+Using a [Raft](https://raft.github.io/raft.pdf) implementation, the
+managers maintain a consistent internal state of the entire swarm and
+all the services running on it. In most cases, one manager should be
+sufficient for a swarm of Engines.
+
+If you require fault-tolerance in case of a manager failure, you should
+maintain an odd number of managers. Docker recommends either three or
+five manager nodes according to your organization's high-availability
+requirements:
+
+* A three-manager swarm tolerates a maximum loss of one manager.
+* A five-manager swarm tolerates a maximum simultaneous loss two
+manager nodes.
+* In general, an `N` manager cluster will tolerate the loss of at most
+`N/2` managers.
+* Seven is the maximum possible number of manager nodes for a swarm.
```

Comment:
I don't think this sentence is correct: 7 is the maximum **recommended** number. But you can go beyond if you want provided that you don't mind the performance cost.